solr/core/src/java/org/apache/solr/response/JSONWriter.java

```diff
@@ -132,11 +133,16 @@ public void writeSolrDocument(String name, SolrDocument doc, ReturnFields return
   //       that the size could not be reliably determined.
   //
 
+  /**
+   * This method will be removed in Solr 9
```

Comment:
do you mean solr 10? we don't need a major version with it deprecated before we can jettison it? folks upgrading from 8.5->9 might be surprised by this.

Reply:
My understanding is that it's OK to change methods across major versions? Unless it's something needed for rolling upgrade or something.